openwisp_users/api/urls.py

```diff
@@ -9,6 +10,24 @@ def get_api_urls(api_views=None):
     urlpatterns = []
     if api_views is None:
         api_views = views
+    urlpatterns += [
+        path('users/organization/', views.organization_list, name='organization_list',),
+        path(
+            'users/organization/<str:pk>/',
+            views.organization_detail,
+            name='organization_detail',
+        ),
+        path('users/user/', views.user_list, name='user_list',),
+        path('users/user/<str:pk>/', views.user_detail, name='user_detail'),
+        path(
+            'users/user/<str:pk>/password/',
+            views.change_password,
+            name='change_password',
+        ),
+        path('users/user/<str:pk>/email/', views.email_update, name='email_update',),
+        path('users/group/', views.group_list, name='group_list'),
+        path('users/group/<str:pk>/', views.group_detail, name='group_detail'),
```

Comment:
Shouldn't this be `users/group/<int:pk>/`, group ids are in integers.

Reply:
Fixed, :+1: